apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Footer/Footer.spec.tsx

```diff
@@ -45,116 +46,67 @@ describe('Footer', () => {
-          variant: 'admin'
-        }}
-      >
-        <Footer />
-      </JourneyProvider>
-    )
-    expect(getByText('Facebook')).toBeInTheDocument()
-  })
-
-  it.skip('should display Host attribute with hosts name if a name is provided', () => {
-    const { getByText } = render(
-      <JourneyProvider
-        value={{
-          journey: {
-            id: 'journeyId',
-            themeMode: ThemeMode.dark,
-            themeName: ThemeName.base,
-            language: {
-              __typename: 'Language',
-              id: '529',
-              bcp47: 'en',
-              iso3: 'eng'
-            },
-            host: {
-              title: `John Geronimo "The Rock" Johnson`
-            }
-          } as unknown as Journey,
-          variant: 'admin'
-        }}
-      >
-        <Footer />
-      </JourneyProvider>
+  it('should render the components', async () => {
+    const { getByText, queryByTestId } = render(
+      <MockedProvider>
+        <SnackbarProvider>
+          <EditorProvider initialState={state}>
+            <TestEditorState />
+            <Footer />
+          </EditorProvider>
+        </SnackbarProvider>
+      </MockedProvider>
     )
 
-    expect(getByText('Hosted by')).toBeInTheDocument()
-    expect(getByText(`John Geronimo "The Rock" Johnson`)).toBeInTheDocument()
-  })
-
-  it.skip('should open property drawer for variant', () => {
-    const dispatch = jest.fn()
-    mockUseEditor.mockReturnValue({
-      state,
-      dispatch
+    await act(async () => {
+      fireEvent.click(getByText('Chat Widget'))
     })
 
-    render(<Footer />)
-
-    expect(dispatch).toHaveBeenCalledWith({
-      type: 'SetSelectedAttributeIdAction',
-      selectedAttributeId: 'hosted-by'
-    })
+    expect(queryByTestId('ChatComponent')).toBeInTheDocument()
+    expect(queryByTestId('HostComponent')).not.toBeInTheDocument()
   })
 
-  it('should open property drawer for chat widget', () => {
-    const { getByText } = render(
+  it('should switch tabs', async () => {
+    const { getByRole } = render(
       <MockedProvider>
         <SnackbarProvider>
           <EditorProvider initialState={state}>
-            <Footer />
             <TestEditorState />
+            <Footer />
           </EditorProvider>
         </SnackbarProvider>
       </MockedProvider>
     )
-    fireEvent.click(getByText('Chat Widget'))
-    expect(getByText('drawerTitle: Chat Widget')).toBeInTheDocument()
+
+    expect(getByRole('tab', { name: 'Hosted By' })).toBeInTheDocument()
+    expect(getByRole('tab', { name: 'Chat Widget' })).toBeInTheDocument()
+
+    fireEvent.click(getByRole('tab', { name: 'Chat Widget' }))
+    expect(getByRole('tab', { name: 'Chat Widget' })).toHaveAttribute(
+      'aria-selected',
+      'true'
+    )
+
+    fireEvent.click(getByRole('tab', { name: 'Hosted By' }))
+    expect(getByRole('tab', { name: 'Hosted By' })).toHaveAttribute(
+      'aria-selected',
+      'true'
+    )
```

Comment:
I think we can simply the assertions here, something like below will be sufficient test coverage for us
```suggestion
    expect(getByRole('tab', { name: 'Hosted By' })).toHaveAttribute(
      'aria-selected',
      'true'
    )

    fireEvent.click(getByRole('tab', { name: 'Chat Widget' }))
    expect(getByRole('tab', { name: 'Chat Widget' })).toHaveAttribute(
      'aria-selected',
      'true'
    )
```